src/amo/components/StaticPages/About.js

```diff
@@ -0,0 +1,180 @@
+              // eslint-disable-next-line react/no-danger
+              dangerouslySetInnerHTML={
+                sanitizeHTML(
+                  i18n.sprintf(
+                    i18n.gettext(`If you have add-on development experience,
+                      %(startLink)sbecome an Add-on Reviewer%(endLink)s! Our reviewers are add-on fans
+                      with a technical background who review add-ons for code quality and stability.`),
+                    { startLink: '<a href="https://wiki.mozilla.org/Add-ons/Reviewers/Applying">', endLink: '</a>' }
+                  ), ['a'])
+              }
+            />
+            <li
+              // eslint-disable-next-line react/no-danger
+              dangerouslySetInnerHTML={
+                sanitizeHTML(
+                  i18n.sprintf(
+                    i18n.gettext(`Help improve this website. It's open source, and you can file bugs
+                      and submit patches. %(github)s contains all of our current bugs, legacy bugs can
+                      still be found in Bugzilla.`),
+                    { github: '<a href="https://github.com/mozilla/olympia/issues">GitHub</a>' }
+                  ), ['a'])
+              }
+            />
+          </ul>
+          <p>{i18n.gettext(`If you're interested in add-ons but not quite as technical, there are still ways to help:`)}</p>
+          <ul>
+            <li>{i18n.gettext('Tell your friends! Let people know which add-ons you use.')}</li>
+            <li
+              // eslint-disable-next-line react/no-danger
+              dangerouslySetInnerHTML={
+                sanitizeHTML(
+                  i18n.sprintf(
+                    i18n.gettext(`Participate in our %(startLink)sforums%(endLink)s.`),
+                    { startLink: '<a href="https://discourse.mozilla-community.org/c/add-ons">', endLink: '</a>' }
+                  ), ['a'])
+              }
+            />
+            <li>
+              {i18n.gettext(`Review add-ons on the site. Add-on authors are more likely to
+                improve their add-ons and write new ones when they know people appreciate their work.`)}
+            </li>
+          </ul>
+        </section>
+        <section>
+          <h2>{i18n.gettext('I have a question')}</h2>
+          <p
+            // eslint-disable-next-line react/no-danger
+            dangerouslySetInnerHTML={
+              sanitizeHTML(
+                i18n.sprintf(
+                  i18n.gettext('A good place to start is our %(startLink)sforums%(endLink)s.'),
+                  { startLink: '<a href="https://discourse.mozilla-community.org/c/add-ons">', endLink: '</a>' }
+                ), ['a'])
+            }
+          />
+          <p
+            // eslint-disable-next-line react/no-danger
+            dangerouslySetInnerHTML={
+              sanitizeHTML(
+                i18n.sprintf(
+                  i18n.gettext(`If you really need to contact someone from the Mozilla team,
```

Comment:
"If you really need to contact someone from the Mozilla team," is quite hostile phrasing. It feels like: "If you *really* need to ask a question, *fiiiiine*." Let's be friendlier 😉

`If you need to contact someone from the Mozilla team, for instance if there is a security issue with this website or you need to report an abusive add-on,  %(startLink)sfind the right person/team to contact here %(endLink)s.`